src/components/App/main.css

```diff
@@ -3,9 +3,9 @@
 div.container-wrapper {
   font-family: 'Montserrat', sans-serif;
   position: absolute;
-  top: 60px;
+  top: 140px;
```

Comment:
needed to change the paddings here because the searchbar + the always-present navbar = 60px + 80px = 140px.